### ISSUE TITLE
[BUG] Accordions Cursor

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#502]
Go to https://modus-web-components.trimble.com/?path=/docs/components-accordion--default

The expand button should have the following CSS

```css
cursor: pointer;
```

See: https://modus-bootstrap.trimble.com/components/accordions/ for example of how it should look.

**Comment by ryan-henness-trimble** [comment]
It may be worth checking other components for cursor changes. I think we’re missing pointer for others as well, it wasn’t a part of the original implementation

**Comment by coliff** [comment]
> It may be worth checking other components for cursor changes. I think we’re missing pointer for others as well, it wasn’t a part of the original implementation

yeah, we're doing a big accessibility audit of all components for all frameworks so I expect I'll be adding many more issues over the coming days/weeks. I think next week I'll see if I can fix some of the issues myself and make PRs for you to review.
